src/Emulator/Tabs/ConfigureTab/DeviceContainer/Device/Device.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "./Device.css";
+
+const Device = ({ deviceName, deviceData }) => {
```

Comment:
please use constants here for `"router-color"`, `"switch-color"`, `"host-color"` at the top of the file.